specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json

```diff
@@ -0,0 +1,583 @@
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The workbook has been successfully deleted."
+          }
+        },
+        "x-ms-examples": {
+          "WorkbookList": {
+            "$ref": "./examples/WorkbookResponseOnly.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroup/{resourceGroupName}/providers/microsoft.insights/workbooks/{resourceName}": {  
+      "put": {
+        "description": "Create a new workbook.",
+        "operationId": "Workbooks_Create",
+        "parameters": [          
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkbookResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "workbookProperties",
+            "description": "Properties that need to be specified to create a new workbook.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Workbook"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The newly created workbook.",
+            "schema": {
+                "$ref": "#/definitions/Workbook"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkbookAdd": {
+            "$ref": "./examples/WorkbookAdd.json"
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates a workbook that has already been added.",
+        "operationId": "Workbooks_Update",
+        "parameters": [          
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkbookResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "workbookProperties",
+            "description": "Properties that need to be specified to create a new workbook.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Workbook"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The workbook definition updated.",
+            "schema": {
+                "$ref": "#/definitions/Workbook"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkbookList": {
+            "$ref": "./examples/WorkbookUpdate.json"
+          }
+        }
+      }
+    },
+    "/providers/microsoft.insights/workbooks/category/{category}": {  

```

Comment:
What is the intent of this API? Can workbook/category live at tenant scope? They seem to be getting created under a resource group.